### DISCUSSION TITLE
设置原生谷歌字体API自动跳转到中科大镜像

### DISCUSSION THREAD

**Discussion by Eterance** [mirai-mamori/Sakurairo#787] [Q&A]
自从发现 cdn4.tianli0.top/fonts 反向锁国区后，打算用谷歌原生的api，反正在国内速度也还行。
![image](https://github.com/mirai-mamori/Sakurairo/assets/25473156/f1d0007d-1eef-41ab-8f0d-1aa29a7b5652)

但是设置之后网页就获取不了字体，查看html发现变成了中科大的镜像：
![image](https://github.com/mirai-mamori/Sakurairo/assets/25473156/1c286f1f-c5db-4103-a23c-ab82b9867e9f)

请问这是个什么情况？有什么办法可以解决吗？

**Comment by KotoriK** [comment]
检查下是不是插件问题 主题是直接使用配置里的url的
